src/arche/report.py

```diff
@@ -58,12 +58,15 @@ def write_rule_outcome(cls, outcome: str, level: Level = Level.INFO) -> None:
     def write_details(self, short: bool = False, keys_limit: int = 10) -> None:
         for result in self.results.values():
             if result.detailed_messages_count:
-                self.write_rule_name(
-                    f"{result.name} ({result.detailed_messages_count} message(s))"
+                display(
```

Comment:
If `display` and `Markdown` are "always" together, maybe a custom function could be helpful

Reply:
I don't see much benefits, the readability is good.
I wanted to think about it once the report is shaped more or less.